curp/src/server/curp_node.rs

```diff
@@ -723,21 +725,24 @@ impl<C: 'static + Command, RC: RoleChange + 'static> CurpNode<C, RC> {
     async fn bcast_vote(curp: &RawCurp<C, RC>, vote: Vote) {
         debug!("{} broadcasts votes to all servers", curp.id());
         let rpc_timeout = curp.cfg().rpc_timeout;
+        let voters = curp.voters();
```

Comment:
Should we hold the voters lock here? The conf might changed before we broadcast vote.

Reply:
Fixed, increase the scope of voter lock

Reply:
I mean maybe we need to forbid apply conf change during the entire vote procedure. Because if the conf is applied during the `handle_vote_resp`, the vote result may be incorrect.

Reply:
It seems impossible to vote or apply conf change at the same time, if a candidate receive an conf change log, this node will become a follower and apply this conf change, the vote response will be ignored.